tests/test_options.py

```diff
@@ -4,7 +4,8 @@
 
 from sqlalchemy import UniqueConstraint, create_engine, Column
 from sqlalchemy.orm import scoped_session, sessionmaker
-from sqlalchemy.exceptions import SQLError, ConcurrentModificationError
```

Comment:
I assume `SQLError` is deprecated in the later version of SQLAlchemy in favor of StatementError?

Reply:
Correct.